Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -268,6 +273,11 @@ local questsToBlacklistBySoDPhase = {
         [90135] = true, -- Hiding Rogue Quick Draw Teldrassil for now as there are too many icons
         [90136] = true, -- Hiding Rogue Quick Draw Durotar for now as there are too many icons
         [90137] = true, -- Hiding Rogue Quick Draw Tirisfal Glades for now as there are too many icons
+        [90139] = true, -- Hiding Warrior Quick Strike Loch Modan for now as there are too many icons
+        [90140] = true, -- Hiding Warrior Quick Strike Elwynn Forest for now as there are too many icons
```

Comment:
```suggestion
        [90140] = true, -- Hiding Warrior Quick Strike Westfall for now as there are too many icons
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5435/commits/a88385b68b5dde20dabc2780fa535cdafdc75b35